tasks/aiida-daemon.yml

```diff
@@ -19,22 +19,22 @@
     daemon_aiida_venv: "{{ aiida_venv |  regex_replace('\\$\\{HOME}', current_user_home) }}"
   when: ansible_service_mgr == "systemd"
 
-- name: Start AiiDA Daemon system service
+- name: Start AiiDA Daemon system service (systemd)
   become: true
   become_user: "{{ root_user }}"
   systemd:
     name: aiida-daemon@{{ aiida_profile_name }}
     enabled: true
     masked: false
     daemon-reload: true
-    state: restarted
+    state: started
```

Comment:
@ltalirz having this as restarted breaks idempotence, was there a reason for this?

Reply:
this might have been added by @espenfl  (?)
in any case, I think it's probably not needed

Reply:
> It appears that on redhat the database cluster always needs to be created.
> On CentOS8:
> 
> ```
> [root@instance /]# systemctl status postgresql.service
> ● postgresql.service - PostgreSQL database server
>    Loaded: loaded (/usr/lib/systemd/system/postgresql.service; enabled; vendor preset: disabled)
>    Active: failed (Result: exit-code) since Sat 2020-12-05 02:54:43 UTC; 48s ago
>   Process: 1291 ExecStartPre=/usr/libexec/postgresql-check-db-dir postgresql (code=exited, status=1/FAILURE)
> 
> Dec 05 02:54:43 instance systemd[1]: Starting PostgreSQL database server...
> Dec 05 02:54:43 instance systemd[1]: postgresql.service: Control process exited, code=exited status=1
> Dec 05 02:54:43 instance systemd[1]: postgresql.service: Failed with result 'exit-code'.
> Dec 05 02:54:43 instance systemd[1]: Failed to start PostgreSQL database server.
> [root@instance /]# journalctl -xe
> -- Unit session-c14.scope has finished starting up.
> -- 
> -- The start-up result is RESULT.
> Dec 05 02:54:42 instance sudo[1269]: pam_unix(sudo:session): session opened for user root by (uid=0)
> Dec 05 02:54:42 instance ansible-ansible.legacy.systemd[1271]: Invoked with name=postgresql state=started enabled=True daem>
> Dec 05 02:54:42 instance systemd[1]: Reloading.
> Dec 05 02:54:43 instance systemd[1]: Starting PostgreSQL database server...
> -- Subject: Unit postgresql.service has begun start-up
> -- Defined-By: systemd
> -- Support: https://access.redhat.com/support
> -- 
> -- Unit postgresql.service has begun starting up.
> Dec 05 02:54:43 instance postgresql-check-db-dir[1291]: Directory "/var/lib/pgsql/data" is missing or empty.
> Dec 05 02:54:43 instance postgresql-check-db-dir[1291]: Use "/usr/bin/postgresql-setup --initdb"
> Dec 05 02:54:43 instance postgresql-check-db-dir[1291]: to initialize the database cluster.
> Dec 05 02:54:43 instance postgresql-check-db-dir[1291]: See /usr/share/doc/postgresql/README.rpm-dist for more information.
> Dec 05 02:54:43 instance systemd[1]: postgresql.service: Control process exited, code=exited status=1
> Dec 05 02:54:43 instance systemd[1]: postgresql.service: Failed with result 'exit-code'.
> Dec 05 02:54:43 instance systemd[1]: Failed to start PostgreSQL database server.
> -- Subject: Unit postgresql.service has failed
> -- Defined-By: systemd
> -- Support: https://access.redhat.com/support
> -- 
> -- Unit postgresql.service has failed.
> -- 
> -- The result is RESULT.
> Dec 05 02:54:43 instance sudo[1269]: pam_unix(sudo:session): session closed for user root
> Dec 05 02:55:10 instance systemd[1]: systemd-logind.service: Got notification message from PID 59, but reception is disable>
> ```

Yes, also it usually needs to be started and enabled.

Reply:
> this might have been added by @espenfl (?)
> in any case, I think it's probably not needed

Honestly I do not remember adding this. But I would not have broken the idempotence if I did not have to. Then again could have been a mistake. Let us go without a restart and correct if needed.